src/dbsync/src/dbsync_implementation.cpp

```diff
@@ -70,10 +70,34 @@ std::shared_ptr<DBSyncImplementation::DbEngineContext> DBSyncImplementation::dbE
     const auto it{ m_dbSyncContexts.find(handle) };
     if (it == m_dbSyncContexts.end())
     {
-        throw dbsync_error
-        {
-            2, "Invalid handle value."
-        };
+        throw dbsync_error { INVALID_HANDLE };
     }
     return it->second;
 }
+
+TXN_HANDLE DBSyncImplementation::createTransaction(const DBSYNC_HANDLE handle,
+                                                   const char** tables,
+                                                   const int /*threadNumber*/,
```

Comment:
we can remove the thread number and the max queue size, they are part of the pipeline context.